### PR TITLE
fix description checksum

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran.alex@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/9143
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -290,8 +290,9 @@ max-allowed-packet = 67_108_864
 # key-path = "/path/to/lightning.key"
 
 # When data importing is complete, tidb-lightning can automatically perform
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-configuration.md:292]:
```suggestion
# When data importing is complete, TiDB Lightning automatically performs
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -290,8 +290,9 @@ max-allowed-packet = 67_108_864
...
-# the Checksum, Compact and Analyze operations. It is recommended to leave
-# these as true in the production environment.
+# the Checksum, Compact and Analyze operations. This configuration is only
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-configuration.md:293]: three configurations, not one
```suggestion
# the Checksum, Compact and Analyze operations. These configurations are only
```

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.4 label because the v6.4 docs have been archived at https://docs-archive.pingcap.com/tidb/v6.4 and will no longer receive new updates.

**Comment by ran-huang** [comment]
closing this because the latest change on master overwrites the change made in this PR

**Pull request closed without merging**